src/launcher.rs

```diff
@@ -396,11 +406,42 @@ impl Launcher {
         Ok(())
     }
 
+    /// Runs the shutdown script if it exists and exits the node-launcher process
+    /// with the exit code returned by the script, otherwise returns 0.
+    fn run_shutdown_script_and_exit(&mut self) -> Result<()> {
+        let exit_code = if Path::new(SHUTDOWN_SCRIPT_PATH).exists() {
+            info!("Running shutdown script at {}.", SHUTDOWN_SCRIPT_PATH);
+            let status = utils::map_and_log_error(
+                Command::new(SHUTDOWN_SCRIPT_PATH).status(),
+                format!("Couldn't execute script at {}", SHUTDOWN_SCRIPT_PATH),
```

Comment:
Nit: I believe we use non-capitalized sentences in logs. Please review other messages and eventually update for consistency's sake.